.github/CODEOWNERS

```diff
@@ -1,20 +1,19 @@
-# general codeowners for all files
-# (Order matters. This needs to be at the top)
-* @nvidia/cccl-codeowners
-
 # Libraries
-thrust/ @nvidia/cccl-thrust-codeowners @nvidia/cccl-codeowners
-cub/ @nvidia/cccl-cub-codeowners @nvidia/cccl-codeowners
-libcudacxx/ @nvidia/cccl-libcudacxx-codeowners @nvidia/cccl-codeowners
+thrust/ @nvidia/cccl-thrust-codeowners
+cub/ @nvidia/cccl-cub-codeowners
+libcudacxx/ @nvidia/cccl-libcudacxx-codeowners
 cudax/ @nvidia/cccl-cudax-codeowners
```

Comment:
Note: by the time we start integrating some components from cudax with cuda.core, we might need Python team here. For now it's OK since I keep an eye on it already.